README.md

```diff
@@ -3,8 +3,8 @@
 ## Table Of Content
 
 * [What is the RedHat Che distribution](#what-is-the-redhat-che-distribution)
-* [RedHat Che distribution Maven build](#redhat-che-distribution-maven-build)
 * [Developer-friendly scripts to build and deploy or rollout to minishift](#developer-friendly-scripts-to-build-and-deploy-or-rollout-to-minishift)
+* [Maven build details](#maven-build-details)
 
```

Comment:
I think that we should have the following sections here:
1. What is the Red Hat Che distribution
2. How to build it
3. How to deploy it
4. Further details

"Developer-friendly scripts to build and deploy or rollout to minishift" is too long and should be splitted in how to build it and how to deploy it.